hardware/scripts/gen_avg.py

```diff
@@ -0,0 +1,61 @@
+import csv
+import os
+import pandas as pd
+import numpy as np
+from collections import deque, defaultdict
+import sys
+
+path = '/scratch2/mempool/hardware/build/traces'
```

Comment:
Do we need to have this hardcoded path in the script? How do you call this python script? Maybe the `path` could be a parameter?

Reply:
No, we don't need to hardcode this, thanks